src/ConfigUpdaterInterface.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\commerce;
+
+/**
+ * Performs configuration updates on behalf of an extension.
+ *
+ * Service to provide methods for extensions to import, revert, or delete
+ * configuration once they have been installed. Intended to be run from
+ * hook_post_update_NAME(), since this involves the modification of entities.
+ *
+ * @see hook_post_update_NAME()
+ */
+interface ConfigUpdaterInterface {
+
+  /**
+   * Imports configuration from extension storage to active storage.
+   *
+   * This action triggers a ConfigRevertInterface::IMPORT event if the
```

Comment:
Let's remove the comments that lie about events being triggered, then discuss events in a followup.